chia/wallet/wallet_node.py

```diff
@@ -189,6 +190,11 @@ async def _start(
             .replace("KEY", db_path_key_suffix)
         )
         path = path_from_root(self.root_path, f"{db_path_replaced}_new")
+        standalone_path = path_from_root(STANDALONE_ROOT_PATH, f"{db_path_replaced}_new")
```

Comment:
This isn't pasting well for me, but here's a suggestion to relocate the "_new" portion of the filename before the file extension:

```suggestion
        db_path_replaced: Path = Path(
            self.config["database_path"]
            .replace("CHALLENGE", self.config["selected_network"])
            .replace("KEY", db_path_key_suffix)
        )
        # Append "_new" before the path extension
        # e.g. "blockchain_wallet_v1_mainnet_1234567890_new.sqlite"
        db_path_replaced = db_path_replaced.with_name(
            f"{db_path_replaced.stem}_new{db_path_replaced.suffix}"
        )
        path = path_from_root(self.root_path, f"{db_path_replaced}")
        standalone_path = path_from_root(STANDALONE_ROOT_PATH, f"{db_path_replaced}")
```

Reply:
Tried to have the suggestion start at line 187 but github isn't having it...

Reply:
Sorry, I should have left the `standalone_path` name with `_new` appended to the end, since that's what we already created for the light wallet betas.